source/draw/gpu/opengl/GraphicBufferYuv420Semiplanar.ooc

```diff
@@ -49,13 +49,13 @@ GraphicBufferYuv420Semiplanar: class extends RasterYuv420Semiplanar {
 		extraRows := Int align(padding, this _stride) / this _stride
 		height := this _size y + this _size y / 2 + extraRows
 		width := this _stride / 4
-		rgbaBuffer := GraphicBuffer new(this buffer handle, IntVector2D new(width, height), width, GraphicBufferFormat Rgba8888, GraphicBufferUsage Texture | GraphicBufferUsage RenderTarget)
+		rgbaBuffer := GraphicBuffer new(this buffer handle, IntVector2D new(width, height), width, GraphicBufferFormat Rgba8888, GraphicBufferUsage Texture | GraphicBufferUsage RenderTarget, false)
```

Comment:
Not in this PR, but I think we can have something better than `false` or `true` arguments in constructors. After some time I always forget the meaning. Maybe we could replace this (and in the ByteBuffer) with enum at some point ?